crates/rattler_package_streaming/src/lib.rs

```diff
@@ -62,6 +63,12 @@ impl From<ZipError> for ExtractError {
     }
 }
 
+impl From<Cancelled> for ExtractError {
```

Comment:
Is this related?

Reply:
```
rattler_package_streaming::seek::get_package_file
``` is a blocking operation ( function ) 

so I'm running it like this:
```
simple_spawn_blocking::tokio::run_blocking_task(move || {
                                rattler_package_streaming::seek::get_package_file::<RunExportsJson>(result)
                            }).await;
                            ```
and the error should be created from `Cancelled`, so I need to implement this conversion